library/src/main/java/com/chuckerteam/chucker/internal/data/repository/HttpTransactionDatabaseRepository.kt

```diff
@@ -5,12 +5,13 @@ import com.chuckerteam.chucker.internal.data.entity.HttpTransaction
 import com.chuckerteam.chucker.internal.data.entity.HttpTransactionTuple
 import com.chuckerteam.chucker.internal.data.room.ChuckerDatabase
 import com.chuckerteam.chucker.internal.support.distinctUntilChanged
-import java.util.concurrent.Executor
-import java.util.concurrent.Executors
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
 
 internal class HttpTransactionDatabaseRepository(private val database: ChuckerDatabase) : HttpTransactionRepository {
 
-    private val executor: Executor = Executors.newSingleThreadExecutor()
+    private val backgroundScope = CoroutineScope(Dispatchers.IO)
```

Comment:
Isn't custom scope with the `IO` dispatcher not needed? `Room` supports `suspend` and `Flow`, so custom dispatcher should not be needed after using suspensions for DAOs.

Reply:
Hmm. Agree. Will fix that